src/QuizController.js

```diff
@@ -89,27 +94,43 @@ class QuizController extends Component {
         })
     }
 
+    handleAgainClick() {
+        this.resetQuiz()
+    }
+
+    handleStartQuizClick() {
+        this.resetQuiz()
+    }
+
     render() {
         const renderedComponent = (
-            this.state.finished
+            this.state.onStartPage
```

Comment:
This nested ternary is not great but I also don't know what would be better here.